submissions/oluwabusolamiEluyela/html/index.html

```diff
@@ -0,0 +1,75 @@
+<!doctype html>
+<html>
+  <head>
+<meta charset "UTF-8">
+  <title>justify.com</title>
+  </head>
+
+  <body>
+    <header>
+      <div> 
+       <h1><a href="#">JUSTIFY</a></h1>
+      </div>
+	 <h2><a href="#">Hot Deals</a>
+         <a href="#">Gift Cards</a>
+         <a href="#">Sell</a><c/enter>
+         <a href="#">Book Rentals</a></h2>
+
+         <li><a href="#">New Releases</a></li><li> <a href="#">General Book Shelf</a></li><li> <a href="#">Today's Deals</a></li><li> 
```

Comment:
Hello Oluwabusiolami, you should begin each li tag on a new line; this makes your codes more readable.

Reply:
Done. Please review